frontend/app/src/App.tsx

```diff
@@ -13,9 +13,13 @@ let exchangeRateInterval: any = null;
 
 function App() {
   // get usd/sat exchange rate every 100 second;
+  const getUserOrganizations = useCallback(async () => {
+    await mainStore.getUserOrganizations();
+  }, [])
 
   useEffect(() => {
     mainStore.getUsdToSatsExchangeRate();
+    getUserOrganizations();
```

Comment:
you should be able to put the getUserOrganization in its own `useEffect` I'm pretty sure since this `useEffect` is for the exchange rate